root/application/src/main/java/at/fhv/itb5c/application/controller/UserFactory.java

```diff
@@ -0,0 +1,55 @@
+package at.fhv.itb5c.application.controller;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import at.fhv.itb5c.application.dto.UserConverter;
+import at.fhv.itb5c.application.dto.UserDTO;
+import at.fhv.itb5c.commons.dto.IUser;
+import at.fhv.itb5c.commons.dto.IUserFactory;
+import at.fhv.itb5c.logging.ILogger;
+import at.fhv.itb5c.model.PersistenceFacade;
+import at.fhv.itb5c.model.entity.User;
+
+public class UserFactory implements IUserFactory, ILogger {
+
+	@Override
+	public IUser createUser() {
+		IUser user = new UserDTO();
+		return user;
+	}
+
+	public IUser save(IUser user) {
+		if (user != null) {
+			User userEntity = UserConverter.toEntity(user);
+
+			userEntity = PersistenceFacade.getInstance().saveOrUpdate(userEntity);
+
+			user = UserConverter.toDTO(userEntity);
```

Comment:
userEntity may be null, because saveOrUpdate returns null if something went wrong.
Null check needed.